src/all/unionmangas/assets/i18n/messages_pt_br.properties

```diff
@@ -0,0 +1,4 @@
+chapters_cannot_be_processed=Capitulos n\u00E3o podem ser processados
```

Comment:
uhh this won't work I think. make sure the `*.properties` files are utf-8 in android studio

https://www.jetbrains.com/help/idea/properties-files.html#1cbc434e